CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Snyk Security - Code and Open Source Dependencies Changelog
 
+## [2.3.6
+### Changes
+- Removed Amplitude telemetry and corresponding setting from VSCode
+
```

Comment:
Is the idea of this PR and #432 to remove everything related to Amplitude?
Would we need to remove also code related to `@amplitude/ampli` and `@amplitude/experiment-node-server`?

Reply:
Experiment service is different from the other analytics, we can still use it for canary/feature flag services.